src/VisualStudio/Core/Def/Packaging/PackageInstallerServiceFactory.cs

```diff
@@ -306,28 +317,32 @@ private void OnSourceProviderSourcesChanged(object sender, EventArgs e)
             bool includePrerelease,
             Guid projectGuid,
             EnvDTE.DTE dte,
-            EnvDTE.Project dteProject)
+            EnvDTE.Project dteProject,
+            IProgressTracker progressTracker,
+            CancellationToken cancellationToken)
         {
             this.AssertIsForeground();
             Contract.ThrowIfFalse(IsEnabled);
 
-            // TODO: consider putting this under a TWD so it would actually be cancellable.
-            var cancellationToken = CancellationToken.None;
+            var description = string.Format(ServicesVSResources.Installing_0, packageName);
```

Comment:
An analyzer exists that would flag this line because you don't provide a `CultureInfo` object as a first argument. I recommend it even if you aren't running the analyzer. Either `CultureInfo.InvariantCulture` (for machine consumption) or `CultureInfo.CurrentCulture` (for human consumption).

Reply:
i don't think we ever provide those for loc strings, so i'm going to consider that out of scope for this change.